examples/selectors/label/README.md

```diff
@@ -0,0 +1,45 @@
+## Label Selector EventListener
+
+Creates an EventListener that serve triggers selected via a label selector.
+
+### Try it out locally:
+
+1. To create the label selector trigger and all related resources, run:
+
+   ```bash
+   kubectl apply -f examples/selectors/label
+   ```
+
+2. Port forward:
+   ```bash
+   kubectl port-forward \
+   -n foo $(kubectl get pod -n foo -o=name \
+   -l eventlistener=listener-label-selector) 8000
+   ```
+
+   **Note**: Instead of port forwarding, you can set the
+   [`serviceType`](https://github.com/tektoncd/triggers/blob/master/docs/eventlisteners.md#serviceType)
+   to `LoadBalancer` to expose the EventListener with a public IP. 
+   For this example, modify `04_eventlistener-label-sel.yaml`
```

Comment:
```suggestion
   For this example, modify `02_eventlistener-label-sel.yaml`
```

Reply:
I think that's my bad -> I opened https://github.com/tektoncd/triggers/pull/1016